src/Validator/IsInt.php

```diff
@@ -103,6 +144,9 @@ public function isValid($value)
 
         if (is_int($value)) {
             return true;
+        } elseif (self::COMPARE_STRICT == $this->strict) {
```

Comment:
Does this need to be an `elseif` here? Doesn't seem connected to the previous statement. Also, should be using a strict comparison (`===`), except if changed to `bool` as above, then it'd just be `if ($this->strict) {`